### DISCUSSION TITLE
Add max-width to container on large screens

### DISCUSSION THREAD

**Discussion by johanneslamers** [craftcms/cms#8376] [Ideas]
### Description
Feature request: Please add max-width to the container. I'm working on 27" monitors. The craftcms control panel is ultra wide then. Hard to scan the page from left to right. It doesn't feel good. Safe button is upper right. Nav is upper left. Profile down left. So much spacing. Bad for UX/AX.

### Solution
````
/* large screen max-width container */
@media only screen and (min-width: 1900px) {

    #container {
        max-width: 1680px;
  	    margin-left: auto;
  	    margin-right: auto;
    }
}
````

### Preview

![schermafdruk 2017-02-01 10 35 39](https://cloud.githubusercontent.com/assets/4696315/22501770/863e7d1e-e86a-11e6-8216-88d06f03cb35.png)

![schermafdruk 2017-02-01 10 37 28](https://cloud.githubusercontent.com/assets/4696315/22501779/8d78f618-e86a-11e6-9d98-c0bfdc8fc7f8.png)

### So isn't it shiny?

**Comment by brandonkelly** [comment]
I'm not 100% opposed to this idea. Thoughts, @benjamindavid ?

**Comment by benjamindavid** [comment]
Not really sure about this one, I feel like it would kind of go against the “app” look and feel we're trying to give to Craft.

@johanneslamers do you have any real world examples in mind?

**Comment by davidhellmann** [comment]
I undertand the author. I'm also on 27" and sometime things are a bit wo wide. But I don't think that's a setting where all happy with it. But I think it's simple to clear with: https://github.com/lindseydiloreto/craft-cpcss @johanneslamers

**Comment by angrybrad** [comment]
I'm with @davidhellmann here. Will never be able to please everyone with this and it can be pulled off with a plugin.

**Comment by matthillco** [comment]
I too find this problematic. Writing in the rich-text editor becomes painful on large screens, because lines end up much too long to parse easily. We know that the optimum reading/writing length is around 80 characters, and it would be great if this could be matched in the UI. 

I'd suggest setting a reasonable maximum width on the fields panel (something like 32em-40em), and then letting the right hand sidebar take up the rest of the space.
